tests/testthat/_snaps/examples.md

```diff
@@ -281,7 +281,7 @@
       pages = {20--23},
       month = {September},
       url = {http://CRAN.R-project.org/doc/Rnews/},
-      pdf = {rnewsRnews_2001-3.pdf},
+      pdf = {http://CRAN.R-project.org/doc/Rnews/Rnews_2001-3.pdf},
```

Comment:
This is now correct, before it just parsed the last part of the String, see https://ctan.javinator9889.com/biblio/bibtex/base/btxdoc.pdf:

Reply:
Thanks for catching the update!